02.create-and-explore/05.avatars/07.customizing-avatar-animations/docs.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Customizing Avatar Animations
+taxonomy:
+    category: docs
+---
+
+By default, avatars in High Fidelity will use the built in standard set of animations.  However, these animations are generic and masculine, which doesn't always fit the character of some avatars, such as dainty ballerinas or cartoon foxes.  It is possible to modify the avatar animations by inserting new animations to replace existing ones.  Additionally, advanced technical animators can adjust the animation state machine and the interpolation and blend parameters for truly unique avatars.
+Here are some guidelines for creating animations in the proper format:
++ Animations must have the High Fidelity standard joint names
++ Animations must have the High Fidelity standard joint orientations (y down the bone) 
++ Keyframes must have keyframes for every joint at the uniform interval of 30 frames per second.
+
+See [this document](https://docs.highfidelity.com/create-and-explore/avatars/avatar-standards) for more information about our avatar standards. 
+
+### Modifying Avatar Animations
+
+This document lists several ways to modify the avatar animations, from the easiest but least flexible, to the hardest but most flexible approach. 
+
+#### Override Animation (Script)
+
+The script call [MyAvatar.overrideAnimation](https://docs.highfidelity.com/api-reference/namespaces/myavatar#.overrideAnimation) can be used to play any animation on the current avatar.  It will smoothly interpolate from the avatar's current pose into the start frame of the provided animation.
+If the animation is hosted remotely, you can use [AnimationCache.prefetch](https://docs.highfidelity.com/api-reference/namespaces/animationcache#.prefetch) to ensure that the animation is in-memory before playback. Otherwise you might see a few frames of t-pose as the animation is downloaded from the remote source.
+This animation will take complete control over all the avatars joints.  Inverse Kinematics of the hands and head for HMD users will be disabled.
+[MyAvatar.restoreAnimation](https://docs.highfidelity.com/api-reference/namespaces/myavatar#.restoreAnimation) can be used to stop the override animation and restore the avatar to its normal set of animations.
+
+#### Override Role Animation (Script)
+
+The default animation state machine has a list of specific animations or  that can be overridden.  The  Roles MyAvatar.getAnimationRoles() script call can be used to get the list of roles for the current avatar.   Each one of these roles can be replaced by a custom animation via the MyAv call.atar.overrideRoleAnimation
```

Comment:
Somethig is wrong with that first sentence.  Was that in the original doc I gave you?

Reply:
Something else is going on with that 3rd sentence MyAv call.atar.override...